### PR TITLE
Logoclick#75

### PR DESCRIPTION
**Pull request by Ademsk1**
Able to click the logo link and it will take you to the homepage, added some tests too.

### CODE REVIEW

File: web/src/AppRouter/AppRouter.test.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-console */
 import React from "react";
-import { fireEvent, render, screen } from "@testing-library/react";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+import userEvent from '@testing-library/user-event'
 import "@testing-library/jest-dom";
```
Comment (DomVinyard) [inline web/src/AppRouter/AppRouter.test.js:5]: some "defined but never used" warnings here, not errors but we should clean them up anyway.

Maybe we should add these to our linter as errors?

File: web/src/AppRouter/AppRouter.test.js [hunk 2]
```diff
@@ -13,9 +14,11 @@ test("Home Page must be the first Page to be rendered", async () => {
 test("navigating from Home page to Topic Page", async () => {
   render(<App />, { wrapper: BrowserRouter });
   const topic = "javascript";
-  const topicLink = screen.getAllByRole("link")[0];
+  const links = screen.getAllByRole("link");
+  const topicLink = links.filter(link=>link.textContent==='javascript')[0]
   const topicName = topicLink.textContent;
```
Comment (DomVinyard) [inline web/src/AppRouter/AppRouter.test.js:19]:
```
Replace `link=>link.textContent==='javascript')[0]` with `(link)·=>·link.textContent·===·"javascript")[0];`
```

This is just a prettier issue, is your prettier set up properly?

File: web/src/Global/Header/Header.jsx [hunk 3]
```diff
@@ -21,7 +22,9 @@ export default function Header() {
   return (
     <div>
       <header>
-        <img src="./explainai-logo.png" alt="logo" />
+        <a href="/" title='Link to homepage'>
+          <img src="./explainai-logo.png" alt="Logo of explain-ai & link to take you back to the homepage" />
```
Comment (DomVinyard) [inline web/src/Global/Header/Header.jsx:26]:
```suggestion
          <img src="./explainai-logo.png" alt="ExplainAI" />
```

Comment (DomVinyard) [reply web/src/Global/Header/Header.jsx:26]: think of alt text as "the text version of this image"

File: web/src/Global/Header/Header.test.js [hunk 4]
```diff
@@ -54,7 +55,7 @@ describe("test if all elements of header are rendered", () => {
   const tags = ["5", "10", "Adult"];
   test("logo rendered", () => {
     render(<Header />);
-    const logo = screen.getByAltText("logo");
+    const logo = screen.getByAltText("Logo of explain-ai & link to take you back to the homepage");
```
Comment (DomVinyard) [inline web/src/Global/Header/Header.test.js:58]:
```suggestion
    const logo = screen.getByAltText("ExplainAI");
```

File: web/src/Global/Header/Header.jsx [hunk 5]
```diff
@@ -21,7 +22,9 @@ export default function Header() {
   return (
     <div>
       <header>
-        <img src="./explainai-logo.png" alt="logo" />
+        <a href="/" title='Link to homepage'>
```
Comment (DomVinyard) [inline web/src/Global/Header/Header.jsx:25]: We should prefer using React Router's `<Link to="">` tag over the `<a>` tag.

https://stackoverflow.com/questions/43087007/react-link-vs-a-tag-and-arrow-function

File: web/src/AppRouter/AppRouter.test.js (hunk 1 above)
Comment (mdapim) [reply web/src/AppRouter/AppRouter.test.js:5]: @DomVinyard Of course I'll clean them up now, would you like me to add the linter rule in this change

File: web/src/AppRouter/AppRouter.test.js (hunk 2 above)
Comment (mdapim) [reply web/src/AppRouter/AppRouter.test.js:19]: I don't think so it's not showing any errors with npm run lint

File: web/src/AppRouter/AppRouter.test.js (hunk 1 above)
Comment (DomVinyard) [reply web/src/AppRouter/AppRouter.test.js]: @mdapim great, thanks - I'll leave that decision to you

**Review by yuna-iwata — APPROVED** [review]
looks good! 👍

File: web/src/Global/Header/Header.css [hunk 6]
```diff
@@ -7,21 +7,29 @@ header {
...
 @media (min-width: 1000.5px) {
-  header img {
+  header a {
     margin-left: 15%;
   }
 }
 @media (max-width: 400.5px) {
-  header img {
-    height: 40%;
+  header a {
```
Comment (Mariam-mohamed110) [inline web/src/Global/Header/Header.css:22]: How comes you switched from using img to a in the code?

**Review by Mariam-mohamed110 — APPROVED** [review]

**Pull request merged**